core/tauri-utils/src/pattern/isolation.js

```diff
@@ -85,8 +103,14 @@
       data = await window.__TAURI_ISOLATION_HOOK__(data)
     }
 
-    const encrypted = await encrypt(data)
-    sendMessage(encrypted)
+    const { cmd, callback, error, payload, options } = data
+    sendMessage({
+      cmd,
+      callback,
+      error,
+      payload: await encrypt(payload),
+      options
+    })
```

Comment:
This should probably be written with `Object.create(null)` like all other objects are created with in the ipc and isolation scripts.
```suggestion
const message = Object.create(null)
message.cmd = data.cmd
message.callback = data.callback
message.error = data.error
message.options = data.options
message.payload = await encrypt(payload)
sendMessage(message)
```

Reply:
nice catch thanks